.swift-version

```diff
@@ -1 +0,0 @@
-4.1
```

Comment:
Why is it dropped? It helps to verify the correct Swift version when you lint the podspec. Right now we have 4.2 so it could've caused issues because it would've tried to build our sources using 4.1.

Reply:
No - it doesn't affect anything.

Reply:
The swift version is specified in podspec - if different one will be used - it will not install the pod.

Reply:
Hmm, but you've dropped it there too